### DISCUSSION TITLE
ColorPicker opacity for HEX colors

### DISCUSSION THREAD

**Discussion by Ruslan-I7** [mantinedev/mantine#4225] [Q&A] [answered]
Hi guys,

I tried to use this part of the code to display both tools, 

`<ColorPicker saturationLabel="Saturation" hueLabel="Hue" alphaLabel="Alpha" />`

but eventually, alphaLabel wasn't shown. My objective is to add `AlphaSlider` for HEX and HEXA and a square that shows a color

![1_339](https://github.com/mantinedev/mantine/assets/57489028/5cc65df4-0d51-4d85-bbec-52f0d8d00d89)

**Comment by rtivital** [comment, marked as answer]
Apha slider and preview swatch are only visible with color formats that support transparency – hexa, rgba and hsla

**Reply by Ruslan-I7** [reply]
Got it, thanks
